### PR TITLE
Updating the index page + other small changes

### PR DESCRIPTION
**Pull request by rivalee**
- Updated the index page with new links 

And small HTML/CSS changes: 
- Hiding the search box (for now)
- Adjusting the size of the blue box

### CODE REVIEW

**Comment by malcolmbaig** [comment]
👍

File: app/assets/sass/design-tabless.scss [hunk 1]
```diff
@@ -632,7 +631,7 @@ width: 60%;
 
 .recent-pubs {
 display: inline-block;
- padding: 10px;
+ padding: 0 0 0 0;
```
Comment (Davidslv) [inline app/assets/sass/design-tabless.scss:634]: you can do `padding: 0;` that will do the same

**Comment by Davidslv** [comment]
👍

**Comment by rivalee** [comment]
@Davidslv Yep! Keeping it hacky and inefficient while still working out what padding works best.

**Pull request merged**